app/page.client.tsx

```diff
@@ -1,11 +1,100 @@
 'use client';
 
-import { Heading } from '@chakra-ui/react';
+import { Center, Flex, Heading, SimpleGrid } from '@chakra-ui/react';
+import PageLink from './_components/pagelink/PageLink';
+import {
+  Attractions,
+  Bodies,
+  Factions,
+  LoopTradeRoute,
+  MultiHopTradeRoute,
+  POIs,
+  Shipyard,
+  SingleTradeRoute,
+  Stations,
+  Systems,
+} from './_icons/modules';
 
 const PageClient = () => (
-  <Heading as="h2" size="lg">
-    Welcome to EDPN!
-  </Heading>
+  <Center width="100%">
+    <Flex flexDirection="column" gap="24px" width="100%" maxWidth="1500px">
+      <Heading as="h1" size="lg">
+        Elite Dangerous Pilots Network
+      </Heading>
+      <SimpleGrid minChildWidth="220px" spacing="50px">
+        <PageLink
+          url="/systems"
+          icon={<Systems />}
+          title="Systems"
+          description="Browse the universe! Jump to any system or search by many properties"
+          notImplemented
+        />
```

Comment:
This is a good use case to make a separate component for this grid view. You can also use the base chakra UI component as I suggested above but run it through a loop and pass it a config of all the links you want to generate. e.g. 

```
const modules = [{title, desc, link, ...}, {}]
```

Keeps the grid component you will create also simple and reduces duplicate code